Ajout devices group

NashobaWellan opened the pull request.

NashobaWellan committed:
[add] afficher device group

Permet d'afficher tout les devices d'un même groupe

NashobaWellan committed:
[modif] afficher les devices dans le bon groupe

Permet d'afficher les devices d'un groupe quand on change de groupes et ne pas garder ceux de l'ancien groupe

sebastienvolont commented on web/src/pages/GroupDetail.tsx:
```diff
@@ -37,6 +41,7 @@ class GroupDetail extends React.Component<GroupProps, Group> {
 
   componentDidMount() {
     this.loadGroup();
+    this.loadDeviceGroup();
```
L'appel de la méthode « this.loadDeviceGroup() » n'est pas nécessaire car celle-ci est appelée dans « loadGroup() ».

sebastienvolont commented on web/src/pages/GroupDetail.tsx:
```diff
@@ -151,6 +166,11 @@ class GroupDetail extends React.Component<GroupProps, Group> {
           </Row>
+          <div>
+            {this.state.devices.map((device: Device, i: number) => (
+              <DeviceBox key={device.Id} name = {device.name} state="success" ipAddress={device.ip} groupId={device.groupId} type={device.type} id={device.id} />
```
La clé du composant DeviceBox affiche un warning dans la console, car celle-ci n'est pas valide.
```key={device.Id}```, l'attribut id doit être écrit en miniscule

sebastienvolont replied:
Lors de la modification d'un appareil sur la page d'un groupe, une erreur apparaît dans la console et ne rafraichit pas l'affichage. 
Il ne trouve pas le props « displayDevice », besoin d'appeler la fonction du chargement du groupe dans le props du composant « DeviceBox ».

sebastienvolont replied:
> Lors de la modification d'un appareil sur la page d'un groupe, une erreur apparaît dans la console et ne rafraichit pas l'affichage. Il ne trouve pas le props « displayDevice », besoin d'appeler la fonction du chargement du groupe dans le props du composant « DeviceBox ».

Donc ici : 
``` displayDevice={this.loadGroup.bind(this)} ```

sebastienvolont reviewed (changes requested):
3 petites modifications à effectuer.

NashobaWellan committed:
Bug fix

- retrait d'un appel de fonction redondant

- appel de la fonction loadGroup pour éviter une erreur lors de la modification d'un appareil
